WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -214,13 +218,28 @@ def create
     permitted_params = [
       :guests,
       :comments,
+      :accepted_at,
+      :deleted_at,
       registration_competition_events_attributes: [:id, :competition_event_id, :_destroy],
     ]
+    params[:registration][:deleted_at] = nil
+    params[:registration][:accepted_at] = nil
     if current_user.can_manage_competition?(competition_from_params)
-      permitted_params << :accepted_at
+      permitted_params << [
+        :accepted_by,
+        :deleted_by,
+      ]
```

Comment:
Hmmm, interesting. It seems to allow accepted_by and deleted_by to be updated. I guess I should use `permitted_params += [ :accepted_by, :deleted_by, ]`